src/main/java/edu/umich/lib/dor/replicaexperiment/service/Replication.java

```diff
@@ -86,7 +86,6 @@ public void execute() {
         sourceRepositoryClient.exportObject(packageIdentifier, objectPathInStaging);
         targetRepositoryClient.importObject(objectPathInStaging);
 
-        targetRepository = repositoryService.getRepository(targetRepositoryName);
```

Comment:
Upon reflection, this re-fetching of the `Repository` instance seemed unnecessary. I might have just missed removing it during earlier refactoring.